### DISCUSSION TITLE
Version 9.5.5

### DISCUSSION THREAD

**Discussion by FilipePS** [FilipePS/Traduzir-paginas-web#462] [Announcements]
- Added:
   - Option to configure four more hotkey settings for chromium browsers. (same options as in Firefox and missing in chromium browsers).

- Changed:
   - Now the extension name is **TWP - Translate Web Pages** in all languages.
   - Disabled the transparent visual style of the translate selected text panel in chromium-based browsers.

- Fixed:
   - Translate Selected Text panel always appears on the left side of the screen in chromium-based browsers.

<hr /><em>This discussion was created from the release <a href='https://github.com/FilipePS/Traduzir-paginas-web/releases/tag/v9.5.5'>Version 9.5.5</a>.</em>

**Comment by MInessota** [comment]
Thank you so much for making this wonderful extension available!!!

**Reply by Kurdish-Techno** [reply]
Please way Kurdish sorani is not in there ?
Please add it❤

**Reply by FilipePS** [reply]
> Please way Kurdish sorani is not in there ? Please add it❤

Will be in version 9.6

**Comment by sduhjh** [comment]
Hi, thank you for developing such a great extension. But I have a little suggestion. I use Chrome. I don't think the current panel is as beautiful as it used to be. The previous panel was more modern and more designed. If you want to make the text easier to read, you can improve on the previous basis, such as increasing the density of the black mask (as it does on Firefox). Of course, everyone's aesthetic is different. In the end it's up to you.
Excuse my machine translation english.
![稿定设计导出-20220902-145343](https://user-images.githubusercontent.com/91155255/188081206-901b698d-4ae7-47d2-ae38-ad65e1dc3365.jpg)

**Comment by FilipePS** [comment]
Version 9.6-beta-1 has been released.
Please test it.
https://github.com/FilipePS/Traduzir-paginas-web/releases/tag/v9.6-beta-1

**Reply by MInessota** [reply]
Hi, thank you very much for a new version!

I can't install the extension in Firefox, it doesn't allow you to install extensions without a signature, I think you have to change the browser's internal settings!

In Vivaldi I could not install the extension either, an error message appears:
"Required value 'version' is missing or invalid. It must be between 1-4 dot-separated integers each between 0 and 65536."
![vivaldi-error](https://user-images.githubusercontent.com/28904791/189741369-af25e72e-5cc4-46f5-a446-813f7cd4c2f1.png)

**Reply by FilipePS** [reply]
Please discuss about version **v9.6-beta-1** in the right place: https://github.com/FilipePS/Traduzir-paginas-web/discussions/473

Apparently Chromium-based browsers don't allow text in the extension version, only numbers.. I already uploaded the extension with the corrected version number.

And to install temporary extensions in Firefox, you need to access this url: `about:debugging`
